package.json

```diff
@@ -21,12 +21,13 @@
   "exports": {
     "./package.json": "./package.json",
     ".": {
+      "types": "./esm/index.d.ts",
```

Comment:
Do you need this? Also, I just noticed that we didn't change other than ".".

Reply:
For some reason esm types throw some errors, that set up the `index.d.ts` as default for everything but internally we use the ones that we had under types on each config